locale-handler.js

```diff
@@ -14,16 +14,8 @@
  * limitations under the License.
  */
 const fs = require("fs");
-const localeCode = require("iso-639-1");
 const path = require("path");
-
-const isProd = Boolean(process.env.GAE_APPLICATION);
-const contentDir = isProd ? "dist" : "src/site/content";
-const dirs = fs.readdirSync(path.join(__dirname, contentDir));
-const SUPPORTED_LOCALES = dirs.filter((dir) => localeCode.validate(dir));
-const DEFAULT_LOCALE = "en";
-
-const isSupported = (locale) => SUPPORTED_LOCALES.indexOf(locale) > -1;
+const locales = require("./src/lib/utils/locale");
```

Comment:
nitpick: can we make this either "locales" the file or "locale" the import? Seems weird to have it be subtly different.